sp/src/game/client/client_hl2.vcxproj

```diff
@@ -866,6 +866,7 @@ exit 1
     <ClInclude Include="toolframework_client.h" />
   </ItemGroup>
   <ItemGroup>
+    <ClCompile Include="chaosmount.cpp" />
```

Comment:
This stuff should be added to `.vpc`, since all of the `.vcxproj` files are temporary and windows only.

`.vcxproj` shouldn't be included in the repository at all, but whatever